nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const app = express();
+
+
+// import data here
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+const reviews = require("./data/reviews");
+//console.log(meals[0].title);
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
```

Comment:
This will make the first test fail, check the homework for how to fix this :) It's the step above the other routes.